src/components/Product/LLMAnalytics/index.tsx

```diff
@@ -0,0 +1,386 @@
+        description: 'Use insights to dive deeper into your data and track sign-ups, payments, and more.',
+        url: '/product-analytics',
+    },
+    {
+        icon: <IconMessage />,
+        product: 'Surveys',
+        description: 'Target users with surveys to gather CSAT scores and measure satisfaction with your model.',
+        url: '/surveys',
+    },
+    {
+        icon: <IconRewindPlay />,
+        product: 'Session replay',
+        description: 'Watch recordings of users interacting with your LLM to get full context about their behavior.',
+        url: '/session-replay',
+    },
+]
+
+export const LLMAnalytics = () => {
+    const { elevenlabs } = useStaticQuery(graphql`
+        fragment ProductCustomerFragment on Mdx {
+            fields {
+                slug
+            }
+            frontmatter {
+                logo {
+                    publicURL
+                }
+                logoDark {
+                    publicURL
+                }
+            }
+        }
+        {
+            elevenlabs: mdx(slug: { eq: "customers/elevenlabs" }) {
+                ...ProductCustomerFragment
+            }
+        }
+    `)
+    const { fullWidthContent } = useLayoutData()
+    return (
+        <>
+            <SEO
+                title="LLM analytics - PostHog"
+                description="Get instant insights into LLM products and model performance"
+                image={`/images/og/surveys.jpg`}
+            />
+            <div className={`${fullWidthContent ? 'max-w-full px-8' : 'max-w-7xl mx-auto'} px-5 py-10 md:pt-20 pb-0`}>
+                <Hero
+                    color="yellow"
+                    icon={<IconAdvanced />}
+                    beta={true}
+                    product={product.capitalized}
+                    title='Unlock <span class="text-red dark:text-yellow">instant insights</span> for LLM products'
+                    description="Track specialist LLM and AI metrics, such as model latency, alongside standard product data"
+                />
+
+                <div className="text-center">
+                    <StaticImage
+                        src="../../../images/products/screenshot-surveys.png"
+                        alt="Screenshot of survey results in PostHog"
+                        className="w-full max-w-[1360px]"
+                        placeholder="none"
+                    />
+                </div>
+                <section id="customers" className="-mt-36 pt-36">
+                    <ul className="list-none p-0 grid md:grid-cols-1 gap-4 mb-10 md:mb-20">
+                        <CustomerCard
+                            outcome="uses the entire PostHog toolset to build their generative voice AI."
+                            quote="We used to have Looker, and a bunch of other tools. Now it's just PostHog, BigQuery, and Stripe...PostHog is amazing. It reins in the chaos so we have everything in one place."
+                            customer={elevenlabs}
```

Comment:
Their name appears as "IIElevanLabs" in the preview. Any idea why?

Reply:
That's their logo!